src/quo2/components/navigation/page_nav/view.cljs

```diff
@@ -30,15 +30,17 @@
              :icon-only?          true
              :size                32
              :on-press            on-press
-             :background          (when (button-properties/backgrounds background) background)
+             :background          (if overlay-shown?
```

Comment:
I feel like this should be 
```
(when background
 (if overlay-shown? :blur background))
```
also I don't think we need the `(button-properties/backgrounds background)` check, seems unnecessary to me 👍

Reply:
Removed the `when` clause as evaluating (#{:foo :bar} :bar) will yield :bar or nil (if key doesn't exist), But I think that we need the check as the button component is used in a lot of places.